src/components/MessagesTile/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Tile } from '../tiles/Tile';
+import { HomeStackScreenProps } from '../../navigators';
+import { useUser, useHasNewMessagesFromUsers } from '../../hooks';
+
+interface Props extends Pick<HomeStackScreenProps<'Home'>, 'navigation'> {}
+
+export function MessagesTile({ navigation }: Props) {
+  const { data } = useUser();
+  const mockUserIds = ['brucetest0005', '6fab714']; // TODO: Fetch from tile config
```

Comment:
Hard-coded for further development until the backend work is finished

Reply:
👍 feels like should be able to resolve this TODO before master merge, the way the backend is shaping up.